RecoLocalMuon/DTSegment/src/DTSegmentUpdator.cc

```diff
@@ -122,16 +125,16 @@ void DTSegmentUpdator::fit(DTRecSegment4D* seg, bool allow3par)  const {
     if(seg->hasZed()) {
 
       // fit in-time Phi segments with the 2par fit and out-of-time segments with the 3par fit
-      if (fabs(seg->phiSegment()->t0())<40.) {
-        fit(seg->phiSegment(),allow3par,0);
```

Comment:
The previous change was an error. I spotted this and reverted to the original version. "block3par" blocks the 3 parameter fit, which is supposed to happen if the segment is "in time". then in line 382 the 2-par fit is executed and it replaces the result of the 3-par fit. 
The cutoff it now configurable via "intime_cut" so if we set it to -1 "block3par" has no effect - this is what is done in the cosmic sequences now.